templates/asp-app.json

```
@@ -174,7 +174,7 @@
       "resources": [
         {
           "name": "[concat('tm', parameters('additional_host_name'))]",
-          "condition": "[not(equals(parameters('is_frontend'), '0'))]",
+          "condition": "[not(contains(createArray('', 'null', 'false', 'localhost', 'debugparam'), parameters('additional_host_name')))]",
```

I don't think localhost and debugparam belong there
```suggestion
          "condition": "[not(contains(createArray('', 'null', 'false'), parameters('additional_host_name')))]",
```